templates/.snapshots/TestRenderAPIGoSuccess-api-api.go.tpl-api-testing.go.snapshot

```diff
@@ -1,5 +1,5 @@
 (*codegen.File)(<nil>
-// Copyright 2022 Outreach Corporation. All Rights Reserved.
+// Copyright 2023 Outreach Corporation. All Rights Reserved.
```

Comment:
[note] unrelated to my changes, but the tests needed to be updated for the new year.